### DISCUSSION TITLE
Resources for Realistic Game Audio

### DISCUSSION THREAD

**Discussion by orcmid** [raysan5/raylib#2912] [General]
I just encountered an outstanding review of a MOOC course focused on the physics of audio for games and ways to achieve realistic audio (effects).    

It is [Physics-Based Sound Synthesis](https://www.classcentral.com/report/review-physics-based-sound-synthesis-stanford/) and seems like a lot of fun.  I am overloaded in projects so I won't dig into this one now.  I wanted to leave a marker though.  

I don't know how well `raudio`/`miniaudio` could be employed for adaptations of these techniques.  

I am curious to know what other solutions/support for realist sound in game worlds folks here recommend or are aware of.

**Comment by raysan5** [comment]
`raudio` is actually quite limited, exposing just basic audio functionality (load and play sound/music) but `miniaudio` exposes way more functionality and even a high-level API. Probably some of those techniques could be implemented using `miniaudio`.
